Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 group :development do
-  gem 'vagrant', git: 'https://github.com/hashicorp/vagrant.git'
+  gem 'vagrant', tag: 'v2.2.4', git: 'https://github.com/hashicorp/vagrant.git'
```

Comment:
Don't we just want latest?

Reply:
No, we want the one we're installing in the Dockerfile.